examples/python/partial_md_example.py

```
@@ -115,8 +115,11 @@ def invalidate_target_metadata(
     ip_addr = "127.0.0.1"
     target_port = args.target_port
     init_port = args.init_port
+    # Listen thread is only needed for socket-based metadata exchange;
+    # when using etcd, metadata is exchanged through etcd instead.
+    enable_listen = not bool(etcd_endpoints)
     # Example using nixl_agent_config
-    agent_config1 = nixl_agent_config(True, True, target_port)
+    agent_config1 = nixl_agent_config(True, enable_listen, target_port)
```

Ruff (0.15.2)

[error] Boolean positional value in function call

(FBT003)